url.bs

```diff
@@ -119,9 +119,12 @@ error.
 <h3 id=percent-encoded-bytes>Percent-encoded bytes</h3>
 
 <p>A <dfn>percent-encoded byte</dfn> is U+0025 (%), followed by two <a>ASCII hex digits</a>.
-Sequences of <a lt="percent-encoded byte">percent-encoded bytes</a>,
-<a for=string>percent-decoded</a>, should not cause <a>UTF-8 decode without BOM or fail</a> to
-return failure.
+
+<p class=note>It is generally a good idea for sequences of <a>percent-encoded bytes</a>, when
```

Comment:
Maybe "It is generally a good idea for sequences of percent-encoded bytes **to be such that**, when ... they do not end up as failure"?

My first reading confused me into thinking this was stating something about the algorithms, like, it's generally a good idea for the algorithms to be constructed such that any input sequence of bytes doesn't fail. But upon re-reading #501 I see the intent now.